components/formats-gpl/src/loci/formats/in/CellH5Reader.java

```diff
@@ -481,8 +495,44 @@ private void parseStructure() throws FormatException {
 
     for (int s=0; s<seriesNames.size(); s++) {
       store.setImageName(seriesNames.get(s), s);
+      
+      String plate_id =  MetadataTools.createLSID("Plate", 0);
+     
+      store.setPlateID(plate_id, 0);
+      store.setPlateName(seriesPlate.get(s), 0);
+      
+      String well_id =  MetadataTools.createLSID("Well", 0);
+      
+      store.setWellID(well_id, 0, 0);
+      
+      String cellh5WellCoord = seriesWell.get(s); 
+      String wellRowLetter = cellh5WellCoord.substring(0, 1);
+      String wellColNumber = cellh5WellCoord.substring(1);
+      
+      int wellRowLetterIndex = "ABCDEFGHIJKLMNOP".indexOf(wellRowLetter);
+      int wellColNumberIndex = -1;
+      try {
+          wellColNumberIndex = Integer.parseInt(wellColNumber);
+      } catch (NumberFormatException e){
+          //
+      }
+     
+      if (wellRowLetterIndex > -1 && wellColNumberIndex > 0) {
+          store.setWellRow(new NonNegativeInteger(wellRowLetterIndex), 0, 0);
+          store.setWellColumn(new NonNegativeInteger(wellColNumberIndex - 1), 0, 0);
+      } else {
+          store.setWellRow(new NonNegativeInteger(0), 0, 0);
+          store.setWellColumn(new NonNegativeInteger(0), 0, 0);
+      }
```

Comment:
Yes, I thought so. Right now, I just write one image. Is it okay to set these indices to zero? Or, how do can I save all open images from ImageJ? With this, I could iterate and increment these indices appropriately.